htdocs/debug/debug.php

```diff
@@ -4,6 +4,8 @@
 
 require_admin_or_401();
 
+LangUtil::setPageId("debug");
```

Comment:
Did this work for you locally? This is definitely the correct line, but it should be _after_ the `require_once(__DIR__."/../includes/header.php");` line, otherwise it completely breaks the page (for me anyway).

The reason for this is that this function sets some global state (the page ID). When the header section is required below, it _also_ calls that function, setting the page ID to something else, and so after that line, the page ID will no longer be "debug". (yay PHP!!1)

Reply:
I followed up in https://github.com/C4G/BLIS/pull/125 since I already merged this one - my bad!